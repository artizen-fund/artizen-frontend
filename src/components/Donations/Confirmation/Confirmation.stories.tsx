import Confirmation from './'

export default {
  title: 'donations/Confirmation',
  component: Confirmation,
  argTypes: {},
}

export const ConfirmationComponent = (props: any) => <Confirmation {...props} />
