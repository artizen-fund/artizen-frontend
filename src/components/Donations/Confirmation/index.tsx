import { useEffect } from 'react'
import styled from 'styled-components'
import { Icon } from '@components'
import { rgba } from '@lib'
import { palette } from '@theme'

const Confirmation = () => {
  return (
    <Wrapper>
      <h1>hey good job</h1>
      <Icon glyph="face" />
    </Wrapper>
  )
}

const Wrapper = styled.div`
  border: 1px dashed ${rgba(palette.uiAlert)};
`

export default Confirmation
