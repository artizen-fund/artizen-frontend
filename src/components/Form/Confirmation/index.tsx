import styled from 'styled-components'
import { rgba } from '@lib'
import { typography, palette } from '@theme'

const Confirmation = styled.div`
  display: none;
  grid-area: confirmation;
  flex-direction: column;
  justify-content: center;
  div {
    ${typography.title.l4}
    color: ${rgba(palette.moon)};
    margin-bottom: 0.25em;
  }
  p {
    ${typography.label.l1}
    color: ${rgba(palette.barracuda)};
  }
  text-align: center;
`
export default Confirmation
