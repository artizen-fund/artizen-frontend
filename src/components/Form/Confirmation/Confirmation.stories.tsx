import Confirmation from './'

export default {
  title: 'components/Confirmation',
  component: Confirmation,
  argTypes: {},
}

export const ConfirmationComponent = (props: any) => (
  <Confirmation {...props}>
    <div>Congrats, confirmation sent!</div>
    <p>Check your email and follow the steps to confirm your subscription.</p>
  </Confirmation>
)
