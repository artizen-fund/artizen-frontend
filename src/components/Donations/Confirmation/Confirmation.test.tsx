import renderer from 'react-test-renderer'
import Confirmation from './'

describe('Confirmation', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Confirmation />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
