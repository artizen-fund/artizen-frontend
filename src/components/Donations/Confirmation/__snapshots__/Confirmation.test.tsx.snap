// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmation renders correctly 1`] = `
<div
  className="sc-7388cd41-0 fhnkiJ"
>
  <h1>
    hey good job
  </h1>
  <div
    className="sc-c089a03b-2 gfJPuX"
  >
    <div
      className="sc-c089a03b-0 eexWtz"
      color="white"
    >
      <div
        className="sc-149d3190-0 izsvxh"
        color="white"
      />
    </div>
  </div>
</div>
`;
